server/routes/incidentDetails/incidentDetails.ts

```diff
@@ -1,190 +1,20 @@
 import { Request, Response } from 'express'
```

Comment:
The 3 original pages are exactly the same now, except for the PageRequestType

Reply:
Not sure if it makes any difference - but could you have a single class and then get the route to construct it with the enum - that way you wouldn't need the three files?

Reply:
That is a decent idea - I might do that later